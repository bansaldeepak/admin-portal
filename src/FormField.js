import React from 'react';
import { StoreProvider } from './lib';
import Field from './Field';

export default (props) => (
  <StoreProvider>
    <Field {...props} />
  </StoreProvider>
);
