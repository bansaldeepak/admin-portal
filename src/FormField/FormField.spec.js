import React from "react";
import { shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import { shallowToJson } from "enzyme-to-json";
import FormField from "./FormField";

configure({ adapter: new Adapter() });

describe("FormField", () => {
  it("Render FormField with default options", () => {
    const component = shallow(<FormField name="form-field" type="text" />);
    const tree = shallowToJson(component);
    expect(tree).toMatchSnapshot();
  });
});
