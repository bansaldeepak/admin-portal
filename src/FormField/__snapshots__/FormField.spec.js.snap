// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormField Render FormField with default options 1`] = `
<StoreProvider>
  <WithStyles(Field)
    name="form-field"
    type="text"
  />
</StoreProvider>
`;
